### PR TITLE
FAQ: clean build re: #306

### PR DESCRIPTION
**Pull request by advayDev1**

### CODE REVIEW

**Comment by confile** [comment]
LGTM

File: FAQ.md [hunk 1]
```diff
@@ -73,6 +73,9 @@ j2objcConfig {
 }
 ```
 
+### Why is my clean build failing?
+This is a [known issue](https://github.com/j2objc-contrib/j2objc-gradle/issues/306).  If you are doing `./gradlew clean build`, try instead `./gradlew clean && ./gradlew build`.  The plugin creates some files during project configuration that the `build` needs but `clean` deletes.
```
Comment (brunobowden) [inline FAQ.md]: Add mention explaining that this doesn't happen when you have any Test.java files and why that is.

**Comment by brunobowden** [comment]
LGTM with comment

File: FAQ.md (hunk 1 above)
Comment (advayDev1) [inline FAQ.md]: done

**Pull request merged**